tox.ini

```diff
@@ -32,7 +32,7 @@ deps =
 commands =
     ; 1. Run Coverage First.
     coverage erase
-    coverage run --include=anomalib/* -m pytest {posargs:tests/pre_merge/} -ra --showlocals
+    coverage run --include=anomalib/* -m pytest {posargs:tests/pre_merge/}
```

Comment:
How does this work now? Should it be `--include=src/anomalib/*`

Reply:
Let me double check